src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -62,8 +63,9 @@ ErrorMessageOr<uint64_t> FindFunctionAddressWithFallback(pid_t pid, std::string_
 [[nodiscard]] ErrorMessageOr<void*> DlopenInTracee(pid_t pid, std::filesystem::path path,
                                                    uint32_t flag) {
   // Make sure file exists.
-  if (!std::filesystem::exists(path)) {
-    return ErrorMessage(absl::StrFormat("File not found (\"%s\")", path));
+  auto file_exists_or_error = orbit_base::FileExists(path);
+  if (file_exists_or_error.has_error() || file_exists_or_error.value() == false) {
+    return ErrorMessage(absl::StrFormat("Library does not exist at: \"%s\"", path));
```

Comment:
in case of a error we lose error_message here.

Reply:
done, see above